docs/spark-structured-streaming.md

```diff
@@ -87,43 +88,39 @@ repartition and sort are considered as heavy operations for streaming workload.
 enable fanout writer to eliminate the requirement.
 
 ```scala
-val tableIdentifier: String = ...
 data.writeStream
     .format("iceberg")
     .outputMode("append")
     .trigger(Trigger.ProcessingTime(1, TimeUnit.MINUTES))
-    .option("path", tableIdentifier)
     .option("fanout-enabled", "true")
     .option("checkpointLocation", checkpointPath)
-    .start()
+    .toTable("database.table_name")
 ```
 
-Fanout writer opens the files per partition value and doesn't close these files till write task is finished.
-This functionality is discouraged for batch query, as explicit sort against output rows isn't expensive for batch workload.
+Fanout writer opens the files per partition value and doesn't close these files till write task is finished. This option is discouraged for batch writing, as explicit sort against output rows is cheap for batch workloads.
 
 ## Maintenance for streaming tables
 
-Streaming queries can create new table versions quickly, which creates lots of table metadata to track those versions.
+Streaming writes can create new table versions quickly, creating lots of table metadata to track those versions.
 Maintaining metadata by tuning the rate of commits, expiring old snapshots, and automatically cleaning up metadata files
 is highly recommended.
 
 ### Tune the rate of commits
 
-Having high rate of commits would produce lots of data files, manifests, and snapshots which leads the table hard
-to maintain. We encourage having trigger interval 1 minute at minimum, and increase the interval if needed.
+Having high rate of commits produces data files, manifests, and snapshots which leads to additional maintenance. We encourage having trigger interval 1 minute at minimum, and increase the interval if needed.
```

Comment:
Nit: I like to avoid personal pronouns like "we" or "I" in documentation because it isn't clear who the speaker is (unlike a blog post, when it is clear that "I" is the author).

Reply:
I fully agree there, thanks!
```suggestion
Having a high rate of commits produces data files, manifests, and snapshots which leads to additional maintenance. It is recommended to have a trigger interval of 1 minute at the minimum and increase the interval if needed.
```